### PR TITLE
Improve code quality

### PR DESCRIPTION
**Pull request by LIUJIAHAOCS**
#121 #118

### CODE REVIEW

**Review by deltango — APPROVED** [review]
Good job updating naming and raising coverage.

File: src/main/java/utask/logic/commands/RelocateCommand.java [hunk 1]
```diff
@@ -31,7 +34,11 @@ public RelocateCommand() {
      * Relocate Command for relocating back to default location.
      * */
     public RelocateCommand(String path, boolean isValidPath) {
-        this.destinationPath = path;
+        if (isValidPath) {
+            this.destinationPath = path;
+        } else {
+            this.destinationPath = "data";
```
Comment (yong24s) [inline src/main/java/utask/logic/commands/RelocateCommand.java:40]: Magic constant = "data"

File: src/main/java/utask/model/util/SampleDataUtil.java [hunk 2]
```diff
@@ -92,6 +118,10 @@ private static String generateDeadline(int i) {
         return fmt.format(calendar.getTime());
     }
 
+    /*
+     * This method is used to random a number for different frequency
+     * make the sample data more realistic
+     */
     private static String randomFrequency() {
```
Comment (yong24s) [inline src/main/java/utask/model/util/SampleDataUtil.java:125]: getRandomFrequencyInString

**Review by yong24s — APPROVED** [review]
Comparator classes are more intuitively named.


Good job.

**Pull request merged**